libs/java/auth_core/src/main/java/com/yahoo/athenz/auth/util/Crypto.java

```diff
@@ -1115,6 +1117,12 @@ public static String extractX509CertSubjectOField(X509Certificate x509Cert) {
         return extractX509CertSubjectField(x509Cert, BCStyle.O);
     }
 
+    public static boolean isRestrictedCertificate(X509Certificate x509Cert) {
```

Comment:
Yeah sounds good. At first I had a single static instance of GlobStringsMatcher in Crypto but it made it hard to unit-test as it required each test to re-initialize the static instance (so it can re-read the property).
I eventually placed it in the method to write the unit-tests and I planned on fixing it by doing the reload in each unit-test before the push but it slipped my mind.
But yeah making the Authority pass the reference is actually a much better solution.